platform/jewel/ui/src/main/kotlin/org/jetbrains/jewel/ui/component/Popup.kt

```diff
@@ -57,17 +62,29 @@ public fun Popup(
     properties: PopupProperties = PopupProperties(),
     onPreviewKeyEvent: ((KeyEvent) -> Boolean)? = null,
     onKeyEvent: ((KeyEvent) -> Boolean)? = null,
+    cornerSize: CornerSize = ZeroCornerSize,
```

Comment:
I know this is experimental, but can we create an overload to avoid breaking changes?